cypress/support/pages/yves/cart/repositories/b2b-cart-repository.ts

```diff
@@ -35,13 +35,21 @@ export class B2bCartRepository implements CartRepository {
     cy.get('.product-quick-add-form__button');
   findCartItemRemovalForm = (sku: string): Cypress.Chainable => {
     return cy.get('[action]').filter((index, element) => {
+      if (Cypress.env('isDynamicStoreEnabled')) {
+        const regex = new RegExp(`^/\\w+/\\w+/cart/async/remove/${sku}/\\w+`);
```

Comment:
Can we use a wildcard css selector instead? That would look cleaner than a regex. Same for all of the below